packages/editor/src/store/actions.js

```diff
@@ -943,6 +959,10 @@ export function* resetEditorBlocks( blocks, options = {} ) {
 		yield* resetLastBlockSourceDependencies( Array.from( updatedSources ) );
 	}
 
+	if ( options.validate ) {
```

Comment:
I considered not making this an option and instead the default behavior. For standard editor usage, we always call `resetEditorBlocks` with `validate: true`. It only really makes sense as an option for usage where a third-party might dispatch this action with the result of a default `wp.blocks.parse` function call, and thus the re-validation would be redundant (note: it would be *redundant*, but not otherwise harmful aside the performance overhead).